emr-dynamodb-hive/src/main/java/org/apache/hadoop/hive/dynamodb/DynamoDBStorageHandler.java

```diff
@@ -237,14 +237,22 @@ public void configureOutputJobProperties(TableDesc tableDesc, Map<String, String
     configureTableJobProperties(tableDesc, jobProperties);
   }
 
+  protected boolean isHiveDynamoDBItemMapType(String type){
```

Comment:
Same here, what is the purpose in wrapping these in another method call?

Reply:
Similar situation where we need to [prefer list type factory](https://github.com/awslabs/emr-dynamodb-connector/pull/71/files#diff-f3a6ef9c1c04a0cfe168d14061163fe3R21)

Reply:
👍